terraform/modules/aws_gundeck-push-notifications/resources.iam.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_iam_user" "gundeck" {
+  name          = "${var.environment}-gundeck-full-access"
+  force_destroy = true
```

Comment:
This is not idempotent, I am not sure we should `force_destroy` (if there's a running gundeck instance it would lose access to AWS). I think we should just fail hard if this fails.

Reply:
I think the naming of that field is a bit misleading, take a look at the [docs](https://www.terraform.io/docs/providers/aws/r/iam_user.html#force_destroy). Should be fine IMHO,, though, i just c/p that from *engagements*

Reply:
> I think the naming of that field is a bit misleading, take a look at the docs. Should be fine IMHO

True, I am confused.

> though, i just c/p that from engagements

Yup, well aware. I think it did recreate the user, I can re-test it 🤔

Reply:
Yes, please do. For now I keep it in there. It's used in different modules. But for each case, we create a user, there is also an access key created. So, at least for those the rule from the docs does not apply. So, I think we could safely omit this flag.

Let me know your findings when re-testing this.